app/controllers/pings_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PingsController < ApplicationController
+  def create
+    ping = Ping.create(ping_params)
+    if ping.persisted?
+      render json: { message: 'Your new trip is now active'}
```

Comment:
Can we fix the formatting to have a space at the end of the curly to match the begining